packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -693,13 +687,13 @@ export abstract class Chart extends Observable implements AgChartInstance {
         return this._axes;
     }
 
-    protected _series: Series[] = [];
-    set series(values: Series[]) {
+    protected _series: Series<any>[] = [];
```

Comment:
Guessing we can't use `Series<unknown>` here and elsewhere since we typically don't care about this parameter at this level of abstraction?

Reply:
I didn't wanna make this PR too big but that's definitely the next step. 
I want the user define `datum` to be `unknown` everywhere, but it'll require a bit more change in the codebase.

Reply:
👍